frontend/packages/console-shared/src/components/virtualized-grid/Grid.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Grid as GridComponent, GridCellProps } from 'react-virtualized';
+import { Item, GridChildrenProps } from './types';
+import { CellMeasurementContext } from './utils';
+import './Grid.scss';
+
+type GridProps = {
+  height: number;
+  width: number;
+  scrollTop: number;
+  items: Item[];
+  children: (props: GridChildrenProps) => React.ReactNode;
+};
+
+const Grid: React.FC<GridProps> = ({ height, width, scrollTop, items, children }) => {
+  const { cache, cellWidth, cellMargin, overscanRowCount, estimatedCellHeight } = React.useContext(
+    CellMeasurementContext,
+  );
+  const itemCount = items.length;
+  const idealItemWidth = cellWidth + cellMargin;
+  const columnCountEstimate = Math.max(1, Math.floor(width / idealItemWidth));
+  const rowCount = Math.ceil(itemCount / columnCountEstimate);
+  const columnCount = Math.max(1, itemCount && Math.ceil(itemCount / rowCount));
+  const cellRenderer = (data: GridCellProps) => children({ data, columnCount, items, rowCount });
+  return (
+    <GridComponent
```

Comment:
Remove tab index from Grid:
```suggestion
    <GridComponent
        tabIndex={null}
```